frontend/src/LogIn.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import Logo from "./logo.png";
+
+// import styled from "styled-components";
+
+export default class LogIn extends React.Component {
+  state = {
+    // as people type in the form, update the state, have the changes reflected in the form
+    email: "",
+    password: "",
+  };
+
+  // change function, accepts e
+  // code for coherency
+  change = (e) => {
+    this.props.onChange({ [e.target.name]: e.target.value });
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  onSubmit = (e) => {
+    e.preventDefault(); // acts with the http address, ensures changes aren't logged to there
+    //this.props.onSubmit(this.state);
+    this.setState({
+      // as people type in the form, update the state, have the changes reflected in the form
```

Comment:
these comments are not super necessarily in my opinon , you can remove them to keep the code clean